### DISCUSSION TITLE
Add Easy & HardRock Mod Customization Options for Difficulty Adjust

### DISCUSSION THREAD

**Discussion by Toasterly** [ppy/osu#25646] [Ideas]
Currently in Lazer selecting Difficulty Adjust has no way of allowing you to play with the values and modifications of HR and EZ by default. I would like to have an option in the Mod Customizations tab to "default" the difficulty adjust values of a map to their Easy or HardRock values for a more convenient time changing the difficulty of each map. 

With the current implementation of Difficulty Adjust you cannot play with the flipped circles of HardRock, and players who play with these mods constantly have to edit the difficulty values of each map they play to their HardRock or Easy mod values before being able to change the value they actually want to.

**Comment by huyenden** [comment]
![image](https://github.com/ppy/osu/assets/128424762/3d2b6fec-d30f-46bf-bfc1-78af53d18b63)

**Reply by Toasterly** [reply]
Yes, I know it is "incompatible" with EZ and HR. I am suggesting it to be made into a mod customization option to default the values of Difficulty Adjust to either their HardRock or Easy values, with an extra option to enable the flipped map for HardRock.

For example: I am an Easy Mod player but I like to play with an Approach Rate even lower than 4-5. When using the Difficulty Adjust mod I have to set all values to their respective EZ values before adjusting the AR to where I'd like it to be. This process has to be repeated for every different map I play.

**Reply by huyenden** [reply]
This means that with the normal ez mod, the ar is divided in half. Maybe you want this ez mod to make the ar divided by 3 times? 🤣 

or hr has a built-in mirror (both)

Although having to adjust the AR coefficient like that is quite tiring, let's wait for ppy's response

**Reply by Toasterly** [reply]
You are misunderstanding what I am asking for entirely,

All I want are the "Default HardRock" & "Default Easy" options in the Difficulty Adjust mod customization menu to make it less tedious for us who play with these values all the time. A bonus to this would be to add a 3rd option to flip the map in the same way HardRock does.

![image](https://github.com/ppy/osu/assets/37195727/6d9e903b-9808-4301-9ee1-17c3be664177)

**Reply by huyenden** [reply]
![image](https://github.com/ppy/osu/assets/128424762/170bff94-09b6-4dfe-bc8e-1290a9bd411e)
Actually if you want to flip patterns in the opposite direction like hardrock, you can use Mirror Vertical, it has the same function as hard rock.

As for the ez mod, I don't really understand it :skull:
